model/security/get_role_mapping/structures.smithy

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetRoleMapping_Input{
+    @required
+    @httpLabel
+    role: String
+}
+
+structure GetRoleMappingStructure{
+    content: RoleMappings
+}
+
+@output
+structure GetRoleMapping_Output {
+    @httpPayload
+    content: GetRoleMappingStructure
```

Comment:
This operation directly returns a map:
```suggestion
@output
structure GetRoleMapping_Output {
    @httpPayload
    content: RoleMappings
```

It is expected that this will cause build errors until #127 is merged